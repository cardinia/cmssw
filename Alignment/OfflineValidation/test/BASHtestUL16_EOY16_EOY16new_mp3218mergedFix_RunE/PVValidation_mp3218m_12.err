27-Sep-2019 21:45:55 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
27-Sep-2019 21:45:58 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root
Begin processing the 1st record. Run 276946, Event 1142658, LumiSection 1 on stream 0 at 27-Sep-2019 21:46:08.465 CEST
Begin processing the 1001st record. Run 276946, Event 9190520, LumiSection 8 on stream 0 at 27-Sep-2019 22:04:03.594 CEST
Begin processing the 2001st record. Run 276946, Event 15899973, LumiSection 14 on stream 0 at 27-Sep-2019 22:06:51.250 CEST
Begin processing the 3001st record. Run 276946, Event 23544395, LumiSection 20 on stream 0 at 27-Sep-2019 22:09:45.599 CEST
Begin processing the 4001st record. Run 276946, Event 32081647, LumiSection 26 on stream 0 at 27-Sep-2019 22:12:40.217 CEST
Begin processing the 5001st record. Run 277071, Event 276088096, LumiSection 148 on stream 0 at 27-Sep-2019 22:13:20.338 CEST
Begin processing the 6001st record. Run 277071, Event 286393191, LumiSection 153 on stream 0 at 27-Sep-2019 22:13:26.811 CEST
Begin processing the 7001st record. Run 277071, Event 293648782, LumiSection 157 on stream 0 at 27-Sep-2019 22:13:33.581 CEST
Begin processing the 8001st record. Run 277071, Event 300473600, LumiSection 161 on stream 0 at 27-Sep-2019 22:13:41.786 CEST
Begin processing the 9001st record. Run 277071, Event 309757126, LumiSection 167 on stream 0 at 27-Sep-2019 22:13:49.403 CEST
Begin processing the 10001st record. Run 277072, Event 812742749, LumiSection 460 on stream 0 at 27-Sep-2019 22:13:58.702 CEST
Begin processing the 11001st record. Run 277072, Event 823629932, LumiSection 465 on stream 0 at 27-Sep-2019 22:14:04.514 CEST
Begin processing the 12001st record. Run 277076, Event 1635584659, LumiSection 1026 on stream 0 at 27-Sep-2019 22:14:09.669 CEST
Begin processing the 13001st record. Run 277076, Event 1643869393, LumiSection 1032 on stream 0 at 27-Sep-2019 22:14:15.357 CEST
Begin processing the 14001st record. Run 277076, Event 1650264355, LumiSection 1037 on stream 0 at 27-Sep-2019 22:14:20.401 CEST
Begin processing the 15001st record. Run 277087, Event 1231121241, LumiSection 1140 on stream 0 at 27-Sep-2019 22:14:25.492 CEST
Begin processing the 16001st record. Run 277087, Event 1234913749, LumiSection 1144 on stream 0 at 27-Sep-2019 22:14:29.091 CEST
Begin processing the 17001st record. Run 277087, Event 1240024066, LumiSection 1148 on stream 0 at 27-Sep-2019 22:14:33.421 CEST
Begin processing the 18001st record. Run 277087, Event 1254257166, LumiSection 1161 on stream 0 at 27-Sep-2019 22:14:37.705 CEST
Begin processing the 19001st record. Run 277087, Event 1258183714, LumiSection 1165 on stream 0 at 27-Sep-2019 22:14:42.999 CEST
Begin processing the 20001st record. Run 277087, Event 1262129404, LumiSection 1169 on stream 0 at 27-Sep-2019 22:14:47.267 CEST
Begin processing the 21001st record. Run 277087, Event 1267742561, LumiSection 1174 on stream 0 at 27-Sep-2019 22:14:51.243 CEST
Begin processing the 22001st record. Run 277087, Event 1272011326, LumiSection 1178 on stream 0 at 27-Sep-2019 22:14:55.748 CEST
Begin processing the 23001st record. Run 277087, Event 1277276461, LumiSection 1182 on stream 0 at 27-Sep-2019 22:14:59.791 CEST
Begin processing the 24001st record. Run 277087, Event 1282597036, LumiSection 1187 on stream 0 at 27-Sep-2019 22:15:03.873 CEST
Begin processing the 25001st record. Run 277087, Event 1286993452, LumiSection 1191 on stream 0 at 27-Sep-2019 22:15:07.074 CEST
Begin processing the 26001st record. Run 277096, Event 2177653683, LumiSection 1248 on stream 0 at 27-Sep-2019 22:15:15.477 CEST
Begin processing the 27001st record. Run 277096, Event 2184812531, LumiSection 1253 on stream 0 at 27-Sep-2019 22:15:21.255 CEST
Begin processing the 28001st record. Run 277096, Event 2193806447, LumiSection 1258 on stream 0 at 27-Sep-2019 22:15:27.432 CEST
Begin processing the 29001st record. Run 277096, Event 2201797314, LumiSection 1264 on stream 0 at 27-Sep-2019 22:15:33.959 CEST
Begin processing the 30001st record. Run 277126, Event 18551719, LumiSection 45 on stream 0 at 27-Sep-2019 22:15:39.076 CEST
Begin processing the 31001st record. Run 277126, Event 26635113, LumiSection 55 on stream 0 at 27-Sep-2019 22:15:46.675 CEST
Begin processing the 32001st record. Run 277202, Event 793183885, LumiSection 506 on stream 0 at 27-Sep-2019 22:15:53.561 CEST
27-Sep-2019 22:15:57 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/F0EE6221-5A07-E911-B26C-002590D9D9FC.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[8.385MB/8.385MB][100%][==================================================][8.385MB/s]  [8.385MB/8.385MB][100%][==================================================][8.385MB/s]  
