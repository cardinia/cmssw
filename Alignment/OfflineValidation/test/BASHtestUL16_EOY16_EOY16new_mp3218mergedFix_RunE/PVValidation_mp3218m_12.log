000 (3196552.012.000) 09/27 21:32:53 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196552.012.000) 09/27 21:45:42 Started transferring input files
	Transferring to host: <188.185.208.15:9618?addrs=188.185.208.15-9618+[2001-1459-301-93--100-9]-9618&noUDP&sock=16446_71d6_56969>
...
040 (3196552.012.000) 09/27 21:45:42 Finished transferring input files
...
001 (3196552.012.000) 09/27 21:45:42 Job executing on host: <188.185.208.15:9618?addrs=188.185.208.15-9618+[2001-1459-301-93--100-9]-9618&noUDP&sock=16313_f08a_3>
...
006 (3196552.012.000) 09/27 21:45:52 Image size of job updated: 235588
	119  -  MemoryUsage of job (MB)
	121404  -  ResidentSetSize of job (KB)
...
006 (3196552.012.000) 09/27 21:50:52 Image size of job updated: 578768
	455  -  MemoryUsage of job (MB)
	465800  -  ResidentSetSize of job (KB)
...
006 (3196552.012.000) 09/27 21:55:52 Image size of job updated: 589620
	466  -  MemoryUsage of job (MB)
	476652  -  ResidentSetSize of job (KB)
...
006 (3196552.012.000) 09/27 22:00:53 Image size of job updated: 612908
	489  -  MemoryUsage of job (MB)
	499940  -  ResidentSetSize of job (KB)
...
006 (3196552.012.000) 09/27 22:05:53 Image size of job updated: 698920
	569  -  MemoryUsage of job (MB)
	582612  -  ResidentSetSize of job (KB)
...
006 (3196552.012.000) 09/27 22:10:54 Image size of job updated: 700036
	571  -  MemoryUsage of job (MB)
	583728  -  ResidentSetSize of job (KB)
...
040 (3196552.012.000) 09/27 22:16:20 Started transferring output files
...
040 (3196552.012.000) 09/27 22:16:20 Finished transferring output files
...
005 (3196552.012.000) 09/27 22:16:20 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:12:16, Sys 0 00:00:06  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:12:16, Sys 0 00:00:06  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	6862  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	6862  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     8631        1   2206850 
	   Memory (MB)          :      571     2000      2000 
...
