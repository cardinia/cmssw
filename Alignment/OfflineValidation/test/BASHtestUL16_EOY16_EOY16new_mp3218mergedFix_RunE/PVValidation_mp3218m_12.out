Job started at  Fri Sep 27 21:45:43 CEST 2019
batch dir: /pool/condor/dir_7062 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_276946_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 8.5M -rw-r--r--. 1 tkello zh 8.4M Sep 27 22:16 PVValidation_mp3218m_276946.root -rw-r--r--. 1 tkello zh 20K Sep 27 21:45 PVValidation_mp3218m_276946_cfg.py -rw-r--r--. 1 tkello zh 5.5K Sep 27 22:16 _condor_stderr -rw-r--r--. 1 tkello zh 521 Sep 27 21:45 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:45 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:45 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:45 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:45 var
Job ended at  Fri Sep 27 22:16:20 CEST 2019
